### ISSUE TITLE
Ne fonctionne plus sur mobile

### ISSUE THREAD

**Issue by astonedf** [AnteJR/UNIL-GameJam#25]

**Comment by AnteJR** [comment]
C'est-à-dire ? L'app ne charge pas ? Les contrôles ne fonctionnent pas ? Normalement je n'ai rien touché qui heurterait le fonctionnement mobile.

**Comment by astonedf** [comment]
Le jeu ne se lance plus, j’ai juste un écran bleu:

![8D15B695-4124-499D-B7B5-1BE7808407CE](https://github.com/AnteJR/UNIL-GameJam/assets/34299252/355db201-1a9e-4687-a7dc-f96b151d2e89)

**Comment by AnteJR** [comment]
Y a une console sur Safari sur iPhone ?

**Comment by loiccattani** [comment]
Oui, il faut le connecter au Mac via un câble USB et ouvrir Safari sur le Mac. Dans le menu développement tu verras l'iPhone et pourras ouvrir une fenêtre pour les outils développeur.

**Comment by astonedf** [comment]
Du coup avec l'iphone connecté j'ai ça comme erreur:
![Screen Shot 2023-10-19 at 08 55 17](https://github.com/AnteJR/UNIL-GameJam/assets/34299252/58af1f6a-f4f5-4731-970f-7244e0a4239e)

**Comment by loiccattani** [comment]
@astonedf En principe tu ne vois pas la page si tu as ouvert la console mobile, il ne devrait y avoir que la console.

![image](https://github.com/AnteJR/UNIL-GameJam/assets/55888/f1cce1f7-1df4-4a47-8882-f50e1b970357)

Ici, Ishtar est mon iPhone, "impossible d'ouvrir la page" est ma page web que je n'arrive pas à ouvrir 🙃

**Comment by astonedf** [comment]
Ouais ça me laisse pas me connecter non plus :/
